typescript/cli/examples/submit/strategy/json-rpc-chain-strategy.yaml

```diff
@@ -1,6 +1,8 @@
 anvil2:
   submitter:
+    chain: anvil2
     type: jsonRpc
```

Comment:
just flagging that the top level key is redundant with the inner key

Reply:
aye, just copying the pattern from the other submitter strategies here